openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -346,16 +346,21 @@ def get_from_id(cls, course_id):
         """
         try:
             course_overview = cls.objects.select_related('image_set').get(id=course_id)
+            if course_overview:
+                log.info("course overview object received for course id: {id}".format(id=course_id))
```

Comment:
@adeelehsan This is going to be very noisy on prod, it would be nice if you can limit this log and on #L353